Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -832,7 +832,6 @@ abbrev NormedCommRing.induced [CommRing R] [NormedRing S] [NonUnitalRingHomClass
 theorem NormOneClass.induced {F : Type*} (R S : Type*) [Ring R] [SeminormedRing S]
     [NormOneClass S] [FunLike F R S] [RingHomClass F R S] (f : F) :
     @NormOneClass R (SeminormedRing.induced R S f).toNorm _ :=
-  -- Porting note: is this `let` a bad idea somehow?
```

Comment:
I'd like a maintainer to double-check this and the next porting note are not actionable/don't track anything useful.